javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -624,7 +624,10 @@ public static boolean hasElementValue(AnnotationMirror anno, CharSequence elemen
                 return expectedType.cast(val.getValue());
             }
         }
-        throw new BugInCF("No element with name \'" + elementName + "\' in annotation " + anno);
+        throw new BugInCF(
```

Comment:
This looks like an unrelated change that should go directly to typetools.

Reply:
It is already in typetools.  This diff shows up here because eisop is behind typetools (and because that typetools change was cherry-picked into this branch).
I have made a pull request: https://github.com/typetools/checker-framework/pull/2645
If you could complete that (I don't think my input is necessary), that would be great.
Once the changes in eisop that are not related to JDK 9 are pull-requested into typetools, we can avoid making painful merges like that one.
Thanks!

Reply:
master should now be in sync with typetools.
@the1derer Can you merge in master and make sure this change disappears?